src/Compilers/CSharp/Portable/Symbols/Metadata/PE/ExternalAnnotations.cs

```diff
@@ -96,5 +145,15 @@ private static void Add(TypeSymbol type, StringBuilder builder)
                         .RemoveMiscellaneousOptions(SymbolDisplayMiscellaneousOptions.UseSpecialTypes)
                         // displaying tuple syntax causes to load the members of ValueTuple, which can cause a cycle, so we use long-hand format instead
                         .WithCompilerInternalOptions(SymbolDisplayCompilerInternalOptions.IncludeNullableReferenceTypeModifier | SymbolDisplayCompilerInternalOptions.UseValueTuple)));
+
+        internal static ImmutableArray<AttributeDescription> GetExtraAttributes(string key)
+        {
+            if (!Attributes.TryGetValue(key, out var descriptions))
```

Comment:
The `if` is not necessary. `return descriptions;` should work regardless of result of `TryGetValue`.